README.md

```diff
@@ -4,7 +4,7 @@
 See the wiki for more information.
 
 ## Installation ##
-`composer require "rosette/api: ~1.5"`
+`composer require "rosette/api: ~1.7"`
```

Comment:
we will call it 1.8 when released right?

Reply:
Yes, although it is just saying that version >= 1.7 and <= 2.0

Reply:
should we stage this to be 1.8 then since it will be required for topics?

Reply:
OK